.circleci/config.yml

```diff
@@ -110,6 +110,7 @@ jobs:
       # kubernetes_test build steps.
       - run: mkdir -p /tmp/logs
       - run: run-go-tests --packages "-p 1 ./..." | tee /tmp/logs/test_output.log
+      - run: run-go-tests --packages "-p 1 -tags=special -run TestRunCommandWithHugeLineOutput ./modules/shell"
```

Comment:
Can you reverse the order so that we run this test first? That way we short circuit and don't run the massive test suite if there is a bug here.